rss2email/email.py

```diff
@@ -360,12 +360,12 @@ def sendmail_send(recipient, message, config=None, section='DEFAULT'):
         p = _subprocess.Popen(
             sendmail + ['-F', sender_name, '-f', sender_addr, recipient],
             stdin=_subprocess.PIPE, stdout=_subprocess.PIPE,
-            stderr=_subprocess.PIPE)
-        stdout,stderr = p.communicate(message_bytes)
+            stderr=_subprocess.STDOUT)
+        stdout, _ = p.communicate(message_bytes)
         status = p.wait()
+        _LOG.error(stdout.decode())
```

Comment:
This should happen only when sendmail actually fails. This being said, it'd probably make sense to have `_LOG.info(stdout.decode())` for cases where sendmail succeeds.

Does that make sense to you?

Reply:
When sendmail is successful there is zero output so there's no reason to suppress sendmail output regardless of the log level. An example:

```sh
$ echo "From: me@myhost.com                                                                                                                                                              
To: user@example.com
Hi, this is my message, and I'm sending it to you!
." | sendmail user@example.com
$ echo $?                                                                                                                                                                                
0
```

Reply:
Well, we'd still display one error line I think:
```python
>>> import logging
>>> logging.error("")
ERROR:root:
```
(also, I wouldn't be surprised if some other sendmail implementations did output things even in successful cases, as sendmail is a “binary” that actually has at least one implementation per SMTP server)

Reply:
Force pushed an update that should resolve this concern.